src/plugins/pcr_primer_design_for_dna_assembly/src/PCRPrimerDesignForDNAAssemblyOPWidget.cpp

```diff
@@ -133,6 +133,9 @@ PCRPrimerDesignForDNAAssemblyOPWidget::PCRPrimerDesignForDNAAssemblyOPWidget(Ann
     connect(tbRightAreaSelectManually, &QAbstractButton::clicked, this, &PCRPrimerDesignForDNAAssemblyOPWidget::sl_selectManually);
     U2WidgetStateStorage::restoreWidgetState(savableWidget);
 
+    tbLeftAreaSelectManually->setChecked(false);
+    tbRightAreaSelectManually->setChecked(false);
```

Comment:
This solutions works, but it's a crutch. The root of the problem isn't here.

`PCRPrimerDesignForDNAAssemblyOPSavableTab` is a class to save state of the tab's elements when the tab closes and restore these states when you open the tab again (`getChildValue` saves the state on closing, `setChildValue` sets the state on opening). There is a possibility to set a custom behavior for some widgets if you want them saving a bit differently. You need to avoid saving state in case of `tbLeftAreaSelectManually` and `tbRightAreaSelectManually`.